packages/code-studio/src/layout/golden-layout.js

```diff
@@ -1,7 +1,7 @@
 import './jquery';
 import './ReactDOM';
 
-import 'golden-layout';
-import 'golden-layout/src/css/goldenlayout-base.css';
-import 'golden-layout/src/css/goldenlayout-dark-theme.css';
+import '@deephaven/golden-layout';
+import '@deephaven/golden-layout/dist/css/goldenlayout-base.css';
+import '@deephaven/golden-layout/dist/css/goldenlayout-dark-theme.css';
 import './GoldenLayout.scss';
```

Comment:
Not your breakage, but noticed while testing:

in lm_header in the GoldenLayout.scss, within codestudio, I think I added overflow: hidden to fix this issue:
![image](https://user-images.githubusercontent.com/1576283/129780127-8293da62-f785-456f-b2a0-27603e335939.png)

where extremely narrow panels icons overflow, but it entirely breaks the tab list dropdown, so I think it's best to remove the overflow: hidden for now.

![image](https://user-images.githubusercontent.com/1576283/129780551-d84b426f-b6a3-445d-b99d-ae4394b6d140.png)

Rest looks fine.

Reply:
I can't think of an easy fix for that with out re-parenting that last in the dom.